### PR TITLE
Build fluent-bit with optimizations

### PR DESCRIPTION
**Pull request by hsmatulisgoogle**
Set cmake flag for building fluent-bit with optimizations, but keeping debug symbols.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -72,7 +72,7 @@ function build_fluentbit() {
   cd submodules/fluent-bit
   mkdir -p build
   cd build
-  cmake .. -DCMAKE_INSTALL_PREFIX=$subagentdir/fluent-bit -DFLB_HTTP_SERVER=On
+  cmake .. -DCMAKE_INSTALL_PREFIX=$subagentdir/fluent-bit -DFLB_HTTP_SERVER=On -DCMAKE_BUILD_TYPE=RELWITHDEBINFO
```
Comment (quentinmit) [inline build.sh:75]: Do you need to do the same thing in Dockerfile.windows?

**Review by quentinmit — APPROVED** [review]

File: build.sh (hunk 1 above)
Comment (hsmatulisgoogle) [reply build.sh:75]: I imagine so, as it looks like the windows build doesn't have debug symbols. I am not familiar with the windows build, do you want me to add that change? (RELEASE -> RELWITHDEBINFO)

Comment (quentinmit) [reply build.sh:75]: Feel free to make that change... It still uses cmake so hopefully the syntax is the same?

Comment (hsmatulisgoogle) [reply build.sh:75]: Done!

**Pull request merged**